src/pages/dev/axelarjs-sdk/tx-status-query-recovery.mdx

```diff
@@ -92,7 +118,7 @@ const response = await sdk.manualRelayToDestChain(
 Possible response values are:
 
 ```ts
-export interface ApproveGatewayResponse {
+interface ApproveGatewayResponse {
```

Comment:
where in the code is this? I know this was in the doc before but I cannot find it in the repo, may have been removed?